src/less/breadcrumb/ds4/breadcrumb.less

```diff
@@ -0,0 +1,16 @@
+@import "../../less/ds4/variables.less";
+@import "../../less/ds4/mixins.less";
+
+@import "../base/breadcrumb.less";
+
+.breadcrumb {
+    li {
+        &:not(:last-of-type)::after {
+            .ds4-icon-chevron-light-right(8px,8px);
+        }
+    }
+
+    a[aria-current] {
+        color: @color-core-gray-dim;
+    }
+}
```

Comment:
High contrast for DS4? (I made them for this purpose. 😉 )

Reply:
Oh, thanks. Didn't know the DS4 ones were ready. Will add.

Reply:
That's why it took me just a tad longer. 👍

Reply:
@seangates Um: `// @ds4-icon-chevron-light-right-base64-light: ''; // no alternative at the moment`

Reply:
Also I notice ds6 is `@ds6-icon-chevron-right-light` and ds4 is `@ds4-icon-chevron-light-right`. Oops. But we can make those the same way around in another PR :-)

Reply:
Crud. There was a reason I didn't do them, and I'm trying to remember. It has something to do with the fact that they weren't in the system before?

Ugh, I can't remember.

Let me make a white fill version, though. That's easy enough. Then you can add it to this PR.

Reply:
```
PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAxMyAyMyI+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTS4yNjMuMjYzYS45Mi45MiAwIDAgMSAxLjI4OCAwTDEyLjA4OCAxMC44Yy4xNzYuMTc2LjI5My40MS4yOTMuNjQ0IDAgLjIzNC0uMTE3LjQ2OC0uMjkzLjY0NEwxLjkwMyAyMi4yNzNjLS4xNzYuMTc2LS40MS4yOTMtLjY0NC4yOTMtLjIzNCAwLS40NjgtLjExNy0uNjQ0LS4yOTNhLjkyLjkyIDAgMCAxIDAtMS4yODhsOS41NDEtOS41NEwuMjYzIDEuNTVhLjkyLjkyIDAgMCAxIDAtMS4yODh6Ii8+PC9zdmc+
```
There you go.

Reply:
Should I just do the rest?

Reply:
Thanks, I'll just drop this one in for now. Feel free to add the rest later.

Reply:
@seangates Sorry, but I now see that DS6 breadcrumb wasn't using the lightweight version of the the chevron right, it uses just the normal one. So I'll need to fix that and get a high contrast version.

Reply:
It appears there is no concept of the lighter weight version in DS6 at all. So I may remove it from DS4 (after checking the impact).

Reply:
Okay. Let me know how I can help.